### PR TITLE
Colaps curl and tar to reduce linecount.

### PR DESCRIPTION
**Pull request by ChristianKniep**
I just like the piping... :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,10 +12,7 @@ ENV SBT_VERSION 0.13.8
 
 # Install Scala
 RUN \
-  cd /root && \
-  curl -o scala-$SCALA_VERSION.tgz http://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz && \
-  tar -xf scala-$SCALA_VERSION.tgz && \
-  rm scala-$SCALA_VERSION.tgz && \
+  curl -fsL http://downloads.typesafe.com/scala/$SCALA_VERSION/scala-$SCALA_VERSION.tgz |tar xfz - -C /root/ && \
```
Comment (hseeberger) [inline Dockerfile]: Please insert a blank btw `|` and `tar`.

Please explain the purpose of the `z` flag and the hyphen before the `-C` flag.

**Comment by hseeberger** [comment]
Why did you close this?

**Comment by ChristianKniep** [comment]
I created a new one... :/

**Comment by ChristianKniep** [comment]
I am able to change an existing PR?

**Comment by hseeberger** [comment]
Sure, mate!

Just do `git push -f`, that's all.

**Comment by ChristianKniep** [comment]
Hehe... you never stop learning in IT.

**Pull request closed without merging**